dashboard/scripts/update_under_13_students.rb

```diff
@@ -6,11 +6,16 @@
 puts "Starting to batch update all students under 13."
 num_students_updated = 0
 
+batch_size = 10000
 min_birthday = Date.today - 13.years
-User.where('birthday IS NULL OR birthday > ?', min_birthday).find_each do |user|
-  user.update! sharing_disabled: true
-  num_students_updated += 1
-  puts "Updated #{num_students_updated} students so far." if num_students_updated % 100000 == 0
+User.where('birthday IS NULL OR birthday > ?', min_birthday).in_batches(of: batch_size) do |where|
```

Comment:
Seems like the variable should be named something like `users` instead of `where`.

Reply:
or `batch`